modules/examples/src/main/java/com/ibm/cloud/platform_services/resource_manager/v2/ResourceManagerExamples.java

```diff
@@ -0,0 +1,180 @@
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.ibm.cloud.platform_services.resource_manager.v2;
+
+import com.ibm.cloud.platform_services.resource_manager.v2.model.CreateResourceGroupOptions;
+import com.ibm.cloud.platform_services.resource_manager.v2.model.DeleteResourceGroupOptions;
+import com.ibm.cloud.platform_services.resource_manager.v2.model.GetQuotaDefinitionOptions;
+import com.ibm.cloud.platform_services.resource_manager.v2.model.GetResourceGroupOptions;
+import com.ibm.cloud.platform_services.resource_manager.v2.model.ListQuotaDefinitionsOptions;
+import com.ibm.cloud.platform_services.resource_manager.v2.model.ListResourceGroupsOptions;
+import com.ibm.cloud.platform_services.resource_manager.v2.model.QuotaDefinition;
+import com.ibm.cloud.platform_services.resource_manager.v2.model.QuotaDefinitionList;
+import com.ibm.cloud.platform_services.resource_manager.v2.model.ResCreateResourceGroup;
+import com.ibm.cloud.platform_services.resource_manager.v2.model.ResourceGroup;
+import com.ibm.cloud.platform_services.resource_manager.v2.model.ResourceGroupList;
+import com.ibm.cloud.platform_services.resource_manager.v2.model.UpdateResourceGroupOptions;
+import com.ibm.cloud.sdk.core.http.Response;
+import com.ibm.cloud.sdk.core.service.exception.ServiceResponseException;
+import com.ibm.cloud.sdk.core.util.CredentialUtils;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+//
+// This file provides an example of how to use the Resource Manager service.
+//
+// The following configuration properties are assumed to be defined:
+// RESOURCE_MANAGER_URL=<service base url>
+// RESOURCE_MANAGER_AUTH_TYPE=iam
+// RESOURCE_MANAGER_APIKEY=<IAM apikey>
+// RESOURCE_MANAGER_AUTH_URL=<IAM token service base URL - omit this if using the production environment>
+// RESOURCE_MANAGER_TEST_QUOTA_ID=<Quota Id used in testing>
+// RESOURCE_MANAGER_TEST_USER_ACCOUNT_ID=<User Id used in testing>
+//
+// These configuration properties can be exported as environment variables, or stored
+// in a configuration file and then:
+// export IBM_CREDENTIALS_FILE=<name of configuration file>
+//
+public class ResourceManagerExamples {
+  private static final Logger logger = LoggerFactory.getLogger(ResourceManagerExamples.class);
+  protected ResourceManagerExamples() { }
+
+  private static String testUserAccountId;
+  private static String testQuotaId;
+  private static String resourceGroupId;
+
+  @SuppressWarnings("checkstyle:methodlength")
+  public static void main(String[] args) throws Exception {
+    ResourceManager service = ResourceManager.newInstance();
+
+    // Load up our test-specific config properties.
+    Map<String, String> config = CredentialUtils.getServiceProperties(ResourceManager.DEFAULT_SERVICE_NAME);
+    testQuotaId = config.get("TEST_QUOTA_ID");
+    testUserAccountId = config.get("TEST_USER_ACCOUNT_ID");
```

Comment:
Also, in the examples we are trying to avoid variable names with the `test` word in it (it might confuse the user who checks the examples in the API docs). Use `example` instead, like `exampleQuotaId`.